packages/@tailwindcss-upgrade/src/codemods/migrate-forms-reset.ts

```diff
@@ -0,0 +1,49 @@
+import dedent from 'dedent'
+import postcss, { type Plugin, type Root } from 'postcss'
+
+const css = dedent
+
+const FORMS_RESET_CSS = css`
+  /*
+    In Tailwind CSS v4, form elements have basic styling applied to them. For
+    compatibility with v3, we've applied the following resets:
```

Comment:
Not sure this is better but maybe this?

```suggestion
    In Tailwind CSS v4, basic styles are applied to form elements by default. To
    maintain compatibility with v3, the following resets have been added:
```